File: dotnet/Lexy/Lexy.Compiler/Compiler/CSharp/BuiltInFunctions/NowFunctionCall.cs
using Lexy.Compiler.Language.Expressions.Functions;
using Lexy.RunTime.RunTime;

namespace Lexy.Compiler.Compiler.CSharp.BuiltInFunctions
{
    internal class NowFunctionCall : NoArgumentFunctionCall
    {
        protected override string ClassName => nameof(BuiltInDateFunctions);
        protected override string MethodName => nameof(BuiltInDateFunctions.Now);

        public NowFunctionCall(NowFunction function) : base(function)
        {
        }
    }
}

File: typescipt/src/compiler/JavaScript/builtInFunctions/nowFunctionCall.ts
import {NoArgumentFunctionCall} from "./noArgumentFunctionCall";
import {NowFunction} from "../../../language/expressions/functions/nowFunction";


export class NowFunctionCall extends NoArgumentFunctionCall {
   protected override className = "BuiltInDateFunctions";
   protected override methodName = "now";

   constructor(functionNode: NowFunction) {
      super(functionNode);
   }
}
